pyccel/codegen/printing/pycode.py

```diff
@@ -458,6 +458,13 @@ def _print_NumpyRandint(self, expr):
             args += ", size = {}".format(size)
         return "randint({})".format(args)
 
+    def _print_NumpyNorm(self, expr):
+        type_name = type(expr).__name__
+        name = type_name[5:].lower()
+        return "{name}({arg})".format(
+                name = name,
+                arg  = self._print(expr.python_arg))
```

Comment:
Can we not print `expr.dim`?

Reply:
because we test only the basic case of numpy.norm `dim` will come `None` which has no printer so `Error at code generation` will be raised. numpy.norm has new additional arguments in the [docs](https://numpy.org/doc/stable/reference/generated/numpy.linalg.norm.html) , and think `keepdim` in this doc can refer to `dim` in Pyccel, So `dim` should not have `None` as a default value.

Reply:
I think `dim` corresponds to `axis` given the effect it has on the shape. It seems to be implemented in fortran so maybe we can add this to the tests? Of course it should only be printed in the cases where `expr.dim is not None`